### PR TITLE
Document Tools & IDEs

### PR DESCRIPTION
**Pull request by praneesha**
- Any checklist items to be verified before production deployment?
- Special styling concerns? 
- Related media files?

### CODE REVIEW

File: website/two-column-pages/docs/learn/vscode-plugin-doc.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+#### Using the VS Code editor
+
+1. Click **View** in the top menu of the editor and click **Command Palette**.
+2. Click **Extensions: Install from VSIX...**, browse and select the VSIX file of the plugin you downloaded.
+![Install using the Command Palette of the editor.](images/command-palette.png)
+
+#### Using the Command Line
+In a new Command Line tab, execute the below command.
+```bash
+$ code --install-extension <BALLERINA-PLUGIN-HOME>
```
Comment (nadeeshaan) [inline website/two-column-pages/docs/learn/vscode-plugin-doc.md]: shall we change <BALLERINA-PLUGIN-HOME> to <BALLERINA-PLUGIN-LOCATION>

File: website/two-column-pages/docs/learn/language-intelligence.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+
+## Workspace symbols
+
+Workspace symbols provide you all the symbols of the current project. These symbols include the services, functions, objects etc. in different the modules of the same project.
+
+> **Tip**: You can access the workspace symbols palette by pressing the Control + T keys.
+
+![Workspace symbols](images/workspace-symbols.gif)
+
+Above are the lanugae intelligence features that are currently available in the Ballerina VS Code Plugin.
```
Comment (nadeeshaan) [inline website/two-column-pages/docs/learn/language-intelligence.md]: Typo _lanugae_

File: website/two-column-pages/docs/learn/intellij-plugin-doc.md [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+
+After installing the IntelliJ Ballerina plugin, you need to [set up Ballerina SDK](set-up-ballerina-sdk.md) for your ballerina projects to activate all capabilities of the plugin. 
+
+## Using the plugin
+
+For information on using the IntelliJ Ballerina plugin to write Ballerina programs, see [Using the IntelliJ Ballerina plugin](#using-intellij-plugin.md)
+
+## Using the features of the plugin
+
+Click on the below links to find information on the various capabilities that are fascilitated by the IntelliJ Ballerina plugin for the development process.
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/intellij-plugin-doc.md]:
```suggestion
Click on the below links to find information on the various capabilities that are facilitated by the IntelliJ Ballerina plugin for the development process.
```

File: website/two-column-pages/docs/learn/intellij-plugin-doc.md [hunk 4]
```diff
@@ -0,0 +1,102 @@
...
+
+- [Running Ballerina programs](using-intellij-plugin-features.md#running-ballerina-programs)
+- [Debugging Ballerina programs](#debugging-ballerina-programs)
+- [Importing modules on the fly](#importing-modules-on-the-fly)
+- [Importing unambiguous modules automatically](#importing-unambiguous-modules-automatically)
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/intellij-plugin-doc.md]:
```suggestion
- [Importing unambiguous modules](#importing-unambiguous-modules)
```

File: website/two-column-pages/docs/learn/intellij-plugin-doc.md [hunk 5]
```diff
@@ -0,0 +1,102 @@
...
+- [Finding usage](#finding-usage)
+- [Formatting Ballerina codes](#formatting-ballerina-codes)
+- [Viewing details of parameters](viewing-details-of-parametyers)
+- [Viewing documentation](#viewing-documentation)
+- [Adding annotation fields via suggestions](#adding-annotation-fields-via-suggestions)
+- [Using file templates](#using-file-templates)
+- [Using live templates and code snippets](#using-live-templates-and-code-snippets)
```
Comment (NipunaRanasinghe) [inline website/two-column-pages/docs/learn/intellij-plugin-doc.md]:
```suggestion
- [Using code snippet templates](#using-code-snippet-templates)
```

File: website/two-column-pages/docs/learn/vscode-plugin-doc.md (hunk 1 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/vscode-plugin-doc.md]: Changed to <BALLERINA-PLUGIN-DIRECTORY>

File: website/two-column-pages/docs/learn/intellij-plugin-doc.md (hunk 3 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/intellij-plugin-doc.md]: Fixed the typo.

File: website/two-column-pages/docs/learn/using-intellij-plugin-features.md [hunk 6]
```diff
@@ -0,0 +1,136 @@
+# Using the IntelliJ plugin features
+
+Click on the below links to find information on the various capabilities that are fascilitated by the IntelliJ Ballerina plugin for the development process.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply website/two-column-pages/docs/learn/using-intellij-plugin-features.md]: Fixed the typo for both IntelliJ and VSCode plugins.

File: website/two-column-pages/docs/learn/intellij-plugin-doc.md [hunk 7]
```diff
@@ -0,0 +1,102 @@
...
+Use either of the below approaches to install the IntelliJ Ballerina plugin.
+
+- [Installing via the IntelliJ IDEA editor](#installing-via-the-intellij-idea-editor)
+- [Installing by downloading the plugin](#installing-by-downloading-the-plugin)
+
+### Installing via the IntelliJ IDEA editor
+
+1. Open the editor, click **IntelliJ IDEA** in the top menu, click **Preferences**, and then click **Plugins**. 
+
+> **Tip:** If you are using Windows, click **File**, click **Settings**, and then click **Plugins**.
```
[the remark this answers is not in the document]
Comment (praneesha) [reply website/two-column-pages/docs/learn/intellij-plugin-doc.md]: Done.

File: website/two-column-pages/docs/learn/intellij-plugin-doc.md (hunk 5 above)
Comment (praneesha) [reply website/two-column-pages/docs/learn/intellij-plugin-doc.md]: Fixed.

**Comment by NipunaRanasinghe** [comment]
@praneesha Shall we resolve the merge conflict as well? :)

**Comment by praneesha** [comment]
> @praneesha Shall we resolve the merge conflict as well? :)

Done.

**Pull request merged**